### PR TITLE
Update kafka config.

### PR DESCRIPTION
**Pull request by jacobcrawford**
The Kafka setup has been updated to version 0.3.3 of the chart.
Resources has been configured for a 3 node setup with VMs that have 4 CPUs and 32Gb memory

### CODE REVIEW

File: yggdrasil/services/kafka/kafka/kafka.yaml [hunk 1]
```diff
@@ -1,14 +1,100 @@
 {{- if .Values.registry -}}
```
Comment (jaxels10) [inline yggdrasil/services/kafka/kafka/kafka.yaml:1]: All the values here will only be set if we set a registry. Should this if statement not be removed?

File: yggdrasil/services/kafka/kafka/kafka.yaml [hunk 2]
```diff
@@ -1,14 +1,100 @@
...
+cluster:
+  kafka: 
+    listeners:
+    - name: internal
+      port: 9092
+      type: internal
+      tls: false
+    storage: 
+      class: ceph-filesystem
```
Comment (jaxels10) [inline yggdrasil/services/kafka/kafka/kafka.yaml:10]: Should we add this to our way of declaring a common storageClass from the yggdrasil values file? So if a storage class is set, template it in? :)

**Review by jaxels10 — CHANGES_REQUESTED** [review]
A couple of questions

File: yggdrasil/services/kafka/kafka/kafka.yaml (hunk 1 above)
Comment (jacobcrawford) [reply yggdrasil/services/kafka/kafka/kafka.yaml:1]: You are right. I guess I did not think about this as it is on main.

**Review by jaxels10 — APPROVED** [review]
This looks real nice

**Pull request merged**